index.bs

```diff
@@ -2244,7 +2260,7 @@ a numbered step. If outdented, it (today) is rendered as a bullet in the midst o
         </dl>
     </li>
 
-1. Throw a "{{NotAllowedError}}" {{DOMException}}. In order to prevent information leak that could identify the
+1. Throw a "{{TimeoutError}}" {{DOMException}}. In order to prevent information leak that could identify the
```

Comment:
I'm not entirely sure we can just change this error type. The point of the caveats in this paragraph is to prevent an information leak via distinguishable error conditions, so changing the type could reintroduce that information leak.

This _might_ be safe now as other pieces of the spec have changed since this was introduced - namely that clients are now expected to ask for user consent before returning an `IllegalStateError`, thus "authorizing" the information leak. But this needs some scrutiny before we can merge the change (and possibly even drop the caveat about the information leak).

Reply:
Well that's unfortunate. If we can't throw `TimeoutError` here then that completely eliminates the possibility of defining a new `TimeoutError`. I'd definitely like to explore this at the next meeting.

Reply:
Yeah, we can probably make this work, we just need to do some due diligence to make sure we're not breaking something. This was written way back in 2017 (commit https://github.com/w3c/webauthn/commit/38e2c4fc25f5622d3a28152951471b15cc81e875 of PR #687), two years before L1, so it's quite likely that it's no longer relevant. Very early on in its life, these procedures were not the asynchronous "wait a while and negotiate with the user before returning a response" we know today, but instead would immediately attempt using any or all currently connected authenticators and then return once all or enough of those attempts were finished. The asynchronous model removes most of the concerns that an "early return" could indicate that a particular credential is present without user consent.

Actually, on review, this may have been mostly redundant [even since that same PR](https://github.com/w3c/webauthn/pull/687#issuecomment-359087435). I think the concern for `makeCredential` was most likely eliminated in PR #809. And I think both operations were already asynchronous even in the early commits of PR #687, but they had very recently become so (part of that change is in #687 itself). Maybe this concern was just because our thinking was still oriented around the semi-synchronous model.

Reply:
From WG Meeting (7/31): We confirmed it's okay to to continue incorporating TimeoutError here. I'll go back over the surrounding language here.

Reply:
Discussion moved to #2132 (please feel free to continue under the assumption that a `TimeoutError` is okay).